File: 1002.py
# The formula to calculate the area of a circumference is defined as A = π * R^R.
# Considering to this problem that π = 3.14159:
#
# Calculate the area using the formula given in the problem description.
#
# Input
# The input contains a value of floating point (double precision), that is the variable R.
#
# Output
# Present the message "A=" followed by the value of the variable, as in the example bellow,
# with four places after the decimal point. Use all double precision variables.
# Like all the problems, don't forget to print the end of line after the result,
# otherwise you will receive "Presentation Error".

pi = 3.14159
r = float(input())

a = pi * r * r

print(f"A={a:.4f}")

File: solutions/1002.cpp
/*
 * Beecrowd 1002 - Area of a Circle
 *
 * Read the radius R (double precision) and compute the circle area
 * A = pi * R^2, using pi = 3.14159 exactly as fixed by the problem. Print
 * "A=<value>" with four digits after the decimal point, then an end-of-line.
 *
 * Input : a floating-point value R.
 * Output: "A=" immediately followed by the area (no spaces around '='),
 *         formatted with four decimals, plus a trailing newline (omitting it
 *         causes a Presentation Error on Beecrowd).
 *
 * Approach: apply the formula with the given constant and print with %.4f.
 *
 * Complexity: O(1) time and space.
 */

#include <cstdio>

int main() {
    double r;
    if (scanf("%lf", &r) != 1) return 0;
    const double PI = 3.14159;          // value fixed by the statement (not M_PI)
    // "A=" has NO spaces around '=' here (unlike problems 1001 / 1003).
    printf("A=%.4f\n", PI * r * r);
    return 0;
}
